src/views/set2/Set2.vue

```diff
@@ -13,21 +13,20 @@
         >
           <br>
           <p class="narrative_text">
-            The USGS and
-            state and local agencies monitor stream temperature across the Delaware
-            River Basin, collecting continuous and discrete measurements of water temperature.
-            In the past 40 years, stream temperature has been measured at more than
-            1,300 sites throughout the basin. Expansion to the monitoring network since 2018
-            is rapidly increasing the volume of available data.
+            At the surface, it appears we have a lot of temperature information in the basin. 
+            The USGS has been monitoring temperature in the DRB since XXXX, and since then has 
+            amassed over XXXXX site-days of temperature measurements.
+          </p>
+          <p class="narrative_text">
+            Since 2017, an uptick in temperature monitoring can be attributed to the USGS’s Next Generation Water 
+            Observing Systems (NGWOS); NGWOS is a targeted effort in the basin to develop and 
+            apply new monitoring techniques for a wide range of water characteristics. Temperature 
+            is one of the most widely measured water characteristics in the U.S. – temperature 
+            can now be monitored by boat, by camera, by drone, by cable, by satellite, by sensor, 
+            or by the tried and true method of a human sticking a thermometer in the water.
           </p>
-          <br>
           <p class="narrative_text">
-            This certainly seems like a lot of information, and it does
-            represent a large investment of resources on the part of the USGS and stakeholders
-            throughout the basin. However, in order to evaluate trends, predict water temperatures,
-            or forecast future water temperatures, we must consider how much information is
-            available to provide as input to a model at any given point in time, and at any
-            given location within the basin.
+            But don’t let the data fool you. Most temperature records are concentrated at a few sites across the basin.
```

Comment:
I'd prefer this statement get a bit modified later as well, since it isn't the data that would fool you, it is the presentation of the data. I think we can make this point without implying either the visual or the data are misleading.

Reply:
Good point - I agree that the current phrasing is problematic